src/locale/ar_SA/LC_MESSAGES/django.po

```diff
@@ -45,654 +45,670 @@ msgstr ""
 
-#: documents/models.py:379
+#: documents/models.py:337
 msgid "has tag"
 msgstr ""
 
-#: documents/models.py:380
+#: documents/models.py:338
 msgid "has any tag"
 msgstr ""
 
-#: documents/models.py:381
+#: documents/models.py:339
 msgid "created before"
 msgstr ""
 
-#: documents/models.py:382
+#: documents/models.py:340
 msgid "created after"
 msgstr ""
 
-#: documents/models.py:383
+#: documents/models.py:341
 msgid "created year is"
 msgstr ""
 
-#: documents/models.py:384
+#: documents/models.py:342
 msgid "created month is"
 msgstr ""
 
-#: documents/models.py:385
+#: documents/models.py:343
 msgid "created day is"
 msgstr ""
 
-#: documents/models.py:386
+#: documents/models.py:344
 msgid "added before"
 msgstr ""
 
-#: documents/models.py:387
+#: documents/models.py:345
 msgid "added after"
 msgstr ""
 
-#: documents/models.py:388
+#: documents/models.py:346
 msgid "modified before"
 msgstr ""
 
-#: documents/models.py:389
+#: documents/models.py:347
 msgid "modified after"
 msgstr ""
 
-#: documents/models.py:390
+#: documents/models.py:348
 msgid "does not have tag"
 msgstr ""
 
-#: documents/models.py:391
+#: documents/models.py:349
 msgid "does not have ASN"
 msgstr ""
 
-#: documents/models.py:392
+#: documents/models.py:350
 msgid "title or content contains"
 msgstr ""
 
-#: documents/models.py:393
+#: documents/models.py:351
 msgid "fulltext query"
 msgstr ""
 
-#: documents/models.py:394
+#: documents/models.py:352
 msgid "more like this"
 msgstr ""
 
-#: documents/models.py:405
+#: documents/models.py:353
+msgid "has tags in"
+msgstr ""
+
+#: documents/models.py:363
 msgid "rule type"
 msgstr ""
 
-#: documents/models.py:409
+#: documents/models.py:365
 msgid "value"
 msgstr ""
 
-#: documents/models.py:415
+#: documents/models.py:368
 msgid "filter rule"
 msgstr ""
 
-#: documents/models.py:416
+#: documents/models.py:369
 msgid "filter rules"
 msgstr ""
 
-#: documents/serialisers.py:53
+#: documents/serialisers.py:64
 #, python-format
 msgid "Invalid regular expression: %(error)s"
 msgstr ""
 
-#: documents/serialisers.py:177
+#: documents/serialisers.py:185
 msgid "Invalid color."
 msgstr ""
 
-#: documents/serialisers.py:451
+#: documents/serialisers.py:459
 #, python-format
 msgid "File type %(type)s not supported"
 msgstr ""
 
 #: documents/templates/index.html:22
-msgid "Paperless-ng is loading..."
+msgid "Paperless-ngx is loading..."
 msgstr ""
 
 #: documents/templates/registration/logged_out.html:14
-msgid "Paperless-ng signed out"
+msgid "Paperless-ngx signed out"
 msgstr ""
 
-#: documents/templates/registration/logged_out.html:45
+#: documents/templates/registration/logged_out.html:59
 msgid "You have been successfully logged out. Bye!"
 msgstr ""
 
-#: documents/templates/registration/logged_out.html:46
+#: documents/templates/registration/logged_out.html:60
 msgid "Sign in again"
 msgstr ""
 
 #: documents/templates/registration/login.html:15
-msgid "Paperless-ng sign in"
+msgid "Paperless-ngx sign in"
```

Comment:
I think these all look good, e.g. this is correct, other stuff is mostly line number changes. Wahoo